src/Sylius/Bundle/AttributeBundle/composer.json

```diff
@@ -26,6 +26,7 @@
         }
     ],
     "require": {
+        "doctrine/doctrine-bundle": "^1.12 || ^2.3.1",
```

Comment:
if you add doctrine bundle on main requirment, add doctrine/orm on main requirement. You cannot put one on dev and one main. it will not enable doctrine orm/